README.md

```diff
@@ -10,6 +10,13 @@ Testing
 Put files to be populated by Ignition in the tree rooted at
 `data/ignition/files/`. Put systemd units in `data/ignition/systemd/units`.
 
+The ZTP manifests provided by the users are read from `data/manifests`.
```

Comment:
Yes, I'll move them to ./manifests for now because they aren't included in the ISO. But eventually we'll need to have the user supplied manifests under /data/ignition/files/ for the to-be-built assisted-service binary to be able to read the manifests and then call the REST-API to create the clusters and infra-env.